src/System.Runtime.WindowsRuntime/src/System/IO/HANDLE_ACCESS_OPTIONS.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
+{
+    [Flags]
+    internal enum HANDLE_ACCESS_OPTIONS : uint
+    {
+        HAO_NONE = 0,
+        HAO_READ_ATTRIBUTES = 0x80,
+        HAO_READ = 0x120089,
+        HAO_WRITE = 0x120116,
+        HAO_DELETE = 0x10000,
+
+        // These are defined elsewhere, adding here for clarity
```

Comment:
I don't get this -- these below don't relate in any obvious way to the HAO_ members above. Eg,.m FILE_READ_DATA != HAO_READ. They're not part of the actual enum and you don't use them so why have them here.